packages/discord-request/src/storage-drivers/memory.ts

```diff
@@ -0,0 +1,23 @@
+import type { StorageDriver } from "../types.js";
+
+export default class MemoryStorageDriver implements StorageDriver {
+	private data: Map<string, any>;
+
+	constructor() {
+		this.data = new Map();
+	}
+
+	async get<T>(key: string, fallback?: T): Promise<T> {
+		const data = this.data.get(key) ?? fallback;
+		return fallback instanceof Number ? Number(data) : data;
```

Comment:
what does this instanceof do? Shouldn't the value already be a number if it was set as one?

Reply:
Right, sorry, careless copy pasting from the redis implementation